blockpage_signatures.json

```diff
@@ -451,131 +451,186 @@
+{"fingerprint":"f_gen_ru_satellite_1","pattern":"<a href=\"http://blocklist\\.rkn\\.gov\\.ru/\""}
+{"fingerprint":"f_gen_se_copyright_satellite","pattern":"blockerad efter ett beslut fr&aring;n Patent"}
+{"fingerprint":"f_gen_sg_1_satellite","pattern":"The High Court of Singapore has ordered a blocking of access to the online location you are attempting to reach as it violates the Copyright Act\\."}
+{"fingerprint":"f_gen_sg_2_satellite","pattern":"The website that you are trying to access is unavailable as it contains prohibited material\\."}
+{"fingerprint":"f_gen_sg_3_satellite","pattern":"pursuant to an Order of the High Court of Singapore"}
+{"fingerprint":"f_gen_siberia_satellite","pattern":"Zakonom je zabranjeno omogućavanje učestvovanja u igrama na sreću koje priređuju neovlašćena lica preko sredstava elektronske komunikacije"}
+{"fingerprint":"f_gen_site_blacklisted_satellite","pattern":"<head><title>This domain has been blacklisted</title></head>"}
+{"fingerprint":"f_gen_site_blocked_satellite","pattern":"<title>Site Blocked</title>"}
+{"fingerprint":"f_gen_swisscom_satellite","pattern":"<title>Website blocked by Swisscom PWLAN Content Filter</title>"}
+{"fingerprint":"f_gen_th_1_satellite","pattern":"बन्देज लगाएको सामग्रीहरु"}
+{"fingerprint":"f_gen_tr_1_satellite","pattern":"aşağıdaki karara istinaden 5651 sayılı Kanun uyarınca Erişim Sağlayıcıları Birliği tarafından erişime engellenmiştir"}
+{"fingerprint":"f_gen_tr_2_satellite","pattern":" sayılı İstanbul Anadolu 14\\.Sulh CM kararıyla erişime engellenmiştir\\.</span>"}
+{"fingerprint":"f_gen_tr_3_satellite","pattern":"SULH CM kararıyla erişime engellenmiştir\\."}
+{"fingerprint":"f_gen_tr_4_satellite","pattern":"sayılı Bakırköy CBS kararıyla erişime engellenmiştir"}
+{"fingerprint":"f_gen_tr_5_satellite","pattern":"sayılı Telekomünikasyon İletişim Başkanlığı kararıyla erişime engellenmiştir"}
+{"fingerprint":"f_gen_tr_6_satellite","pattern":"has been blocked by the decision dated "}
+{"fingerprint":"f_gen_tw_1_satellite","pattern":"<title> TANWP不當資訊防護系統 - 您疑似欲連線至不當網站</title>"}
+{"fingerprint":"f_gen_tw_2_satellite","pattern":"<title> KPPRC惡意網站防護系統 - 您疑似欲連線至惡意網站</title>"}
+{"fingerprint":"f_gen_tw_redirect_302_1_satellite","pattern":" <a href=\"https://twb\\.moe\\.edu\\.tw/deny\\.html\""}
+{"fingerprint":"f_gen_uk_redirect_302_satellite","pattern":"Location: http://www\\.ukispcourtorders\\.co\\.uk"}
+{"fingerprint":"f_gen_website_blocked_2_satellite","pattern":"<title>Website Blocked</title>"}
+{"fingerprint":"f_gen_website_blocked_satellite","pattern":"<title>WEBSITE BLOCKED</title>"}
+{"fingerprint":"e_unk_redirect_301_data_lounge_satellite","pattern":"Location: http://www\\.datalounge\\.com/"}
+{"fingerprint":"c_isp_ru_ttk_redirect_302_satellite","pattern":"Location: http://fz139\\.ttk\\.ru"}
+{"fingerprint":"f_gen_ru_redirect_302_2_satellite","pattern":"Location: http://vasexperts\\.ru/test/blocked\\.php"}
```

Comment:
Vasexperperts should be in prod, it should already have an entry

Reply:
Fixed